Sources/MongoSwift/ConnectionString.swift

```diff
@@ -40,6 +40,22 @@ internal class ConnectionString {
             }
         }
 
+        if let heartbeatFrequencyMS = options?.heartbeatFrequencyMS {
+            guard heartbeatFrequencyMS >= 500 && heartbeatFrequencyMS <= Int32.max else {
```

Comment:
ditto comment in #509 about doing the cast in the `guard`

Reply:
aha, I was thinking I couldn't do it because I need to compare against 500 too, but I forgot I can one-line it with
`guard let value = Int32(exactly: heartbeatFrequencyMS), value >= 500 else { ...`

done